routes/api/contacts.js

```diff
@@ -8,12 +8,25 @@ router.get("/", authMiddleware, contactsController.getAll);
 
 router.get("/:contactId", contactsController.getOne);
 
-router.post("/", authMiddleware, validation.creatingContact, contactsController.addOne);
+router.post(
+  "/",
+  authMiddleware,
+  validation.creatingContact,
+  contactsController.addOne
+);
```

Comment:
To zaczyna wygladac malo czytelnie (i powtarza sie). Mysle, ze moglbys smialo zrobic destrukturyzacje na gorze pliku dla `validation` i `contactsController` (tym bardziej, ze plik nazywa sie `contacts.js`, wiec mala szansa, ze bedziesz tutaj trzymal inne routes), i wtedy robic:
`router.post("/", authMiddleware, creatingContact, addOne)`

Jezeli `creatingContact` jest semantycznie za malo jasne, to mozesz je zmienic np. na `validateCreation` albo (zeby dopasowac je do `addOne`, ktore masz) `validateAddition` (lub odwrotnie: przerob `addOne` na `createOne`).